gateway-server/pom.xml

```diff
@@ -409,6 +409,19 @@
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
         </dependency>
+	<dependency>
+            <groupId>de.thetaphi</groupId>
+            <artifactId>forbiddenapis</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+            <version>0.1.54</version>
```

Comment:
No versions should be listed here at all. Versions should only be specified in the top level pom. Also there is formatting issues here.